### ISSUE TITLE
 report “anchor-is-valid" errors

### ISSUE THREAD

**Issue by cjzy-1** [strapi/foodadvisor#54]
step 3,yarn start,In CardSection/index.js  Line 49:  and About/index.js Line 92:  report “anchor-is-valid" errors ,"The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid"

**Comment by vcoisne** [comment]
@Mcastres did you see this ?

**Comment by Mcastres** [comment]
Yes, I will change it to a button in the next update :)
